recipes/gettext/all/conanfile.py

```diff
@@ -0,0 +1,127 @@
+        libiconv_prefix = tools.unix_path(libiconv_prefix) if tools.os_info.is_windows else libiconv_prefix
+        libxml2_prefix = tools.unix_path(libxml2_prefix) if tools.os_info.is_windows else libxml2_prefix
+        args = ["HELP2MAN=/bin/true",
+                "EMACS=no",
+                "--disable-nls",
+                "--disable-dependency-tracking",
+                "--enable-relocatable",
+                "--disable-c++",
+                "--disable-java",
+                "--disable-csharp",
+                "--disable-libasprintf",
+                "--disable-curses",
+                "--with-libiconv-prefix=%s" % libiconv_prefix,
+                "--with-libxml2-prefix=%s" % libxml2_prefix]
+        build = None
+        host = None
+        rc = None
+        if self.options.get_safe("shared"):
+            args.extend(["--disable-static", "--enable-shared"])
+        else:
+            args.extend(["--disable-shared", "--enable-static"])
+        if self._is_msvc:
+            # INSTALL.windows: Native binaries, built using the MS Visual C/C++ tool chain.
+            build = False
+            if self.settings.arch == "x86":
+                host = "i686-w64-mingw32"
+                rc = "windres --target=pe-i386"
+            elif self.settings.arch == "x86_64":
+                host = "x86_64-w64-mingw32"
+                rc = "windres --target=pe-x86-64"
+            automake_perldir = os.getenv('AUTOMAKE_PERLLIBDIR')
+            if automake_perldir.startswith('/mnt/'):
+                automake_perldir = automake_perldir[4:]
+            args.extend(["CC=%s/compile cl -nologo" % automake_perldir,
+                         "LD=link",
+                         "NM=dumpbin -symbols",
+                         "STRIP=:",
+                         "AR=%s/ar-lib lib" % automake_perldir,
+                         "RANLIB=:"])
+            if rc:
+                args.extend(['RC=%s' % rc, 'WINDRES=%s' % rc])
+        with tools.vcvars(self.settings) if self._is_msvc else tools.no_op():
+            with tools.environment_append(VisualStudioBuildEnvironment(self).vars) if self._is_msvc else tools.no_op():
+                with tools.chdir(os.path.join(self._source_subfolder, self._gettext_folder)):
+                    env_build = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
+                    if self._is_msvc:
+                        env_build.flags.append("-FS")
+                    env_build.configure(args=args, build=build, host=host)
+                    env_build.make(self._make_args)
+
+    def package(self):
+        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        suffix = ".exe" if self.settings.os_build == "Windows" else ""
+        for executable in ["gettext", "ngettext"]:
+            executable += suffix
+            self.copy(pattern=executable, dst="bin", src=os.path.join(self._source_subfolder, self._gettext_folder, "src"), keep_path=False)
```

Comment:
Remove the libraries, if some are packaged?

Reply:
do you mean run `make install` and remove libs and headers?

Reply:
Yes, or at least make sure they are removed in `self.package_info`